integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r4/EncounterFhirResourceProviderIntegrationTest.java

```diff
@@ -106,6 +114,62 @@ public void shouldReturnExistingEncounterAsJson() throws Exception {
 		assertThat(encounter, validResource());
 	}
 	
+	@Test
+	public void shouldPatchExistingMedicationViaJsonMergePatch() throws Exception {
```

Comment:
Check the naming of functions. These should also be immediately after the `update` tests.